openvalidators/reward/nsfw.py

```diff
@@ -37,26 +37,30 @@ def __init__( self, device: str ):
 
     def reward( self, prompt: str, completion: str, name: str ) -> float:
```

Comment:
We should use a consistent API for all reward methods. Right the the design is inconsistent, two options are:

1. Passing bound as an optional argument in all `reward` methods
2. Defining bound in config (since the user doesn't change the number during runtime anyway)